### PR TITLE
Created Dockerfile & Python File for the Application

### PR DESCRIPTION
**Pull request by RohanRusta21**
Hi @Milind220 @Miller2014 @lahdjirayhan @AliShahpurwala @Tomiiwa @ShootGan ,

I have created Dockerfile & Respective python code files which helps us the application to be get executed using cli commands. I have used Docker so that anyone can easily spin up the API in light weight containers without worrying about the installations of any external & 3rd part libraries. I hope this contribution is useful and valuable for the application. Also I have updated the README.md file with the steps so that anyone can easily follow them and use the application. 

Please Review & Approve it :)

Thanks,
Rohan

### CODE REVIEW

**Comment by Milind220** [comment]
@RohanRusta21 Hey Rohan, I've reviewed your PR. Unfortunately, I don't think that dockerizing Ozon3, which is intended to be used simply as a Python library, is a good idea or helpful to the people who use it. As such I've decided to not merge your PR.

Thank you anyways for putting in the effort to make the PR.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Use an official Python runtime as a parent image
+FROM python:3.8
+MAINTAINER label="Rohan Rustagi"
```
Comment (Milind220) [inline Dockerfile:3]: LABEL maintainer="Rohan Rustagi" would be the modern way to write this

Comment (RohanRusta21) [reply Dockerfile:3]: Thanks for the feedback.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy your Python script into the container
+COPY *.py .
```
Comment (Milind220) [inline Dockerfile:8]: This wouldn't scale well with projects as they grow, as it indiscriminately copies all files from the build context into the docker image

**Pull request closed without merging**